src/resources/UsageAnalytics/Read/DataHealth/tests/DataHealth.spec.ts

```diff
@@ -125,4 +126,21 @@ describe('DataHealth', () => {
             );
         });
     });
+
+    describe('getFailedInstances', () => {
+        it('should make a GET call to the data health get failed instances url', async () => {
+            const datahealthGetFailedInstancesParams: DataHealthGetFailedInstancesParams = {
+                ...baseParams,
+                criterionId: 'criterion',
+                group: 'group',
+                scopeValue: '',
+                trackingId: [''],
```

Comment:
Since those are optional params, could you just omit them instead of passing an empty string and an array containing an empty string?